app/models/work.rb

```diff
@@ -26,7 +26,7 @@ class Work < ApplicationRecord
     world: 'world'
   }
 
-  state_machine initial: :first_draft do
+  state_machine initial: :initial do
```

Comment:
Can you regenerate the state machine diagrams as part of this PR?

Per https://github.com/sul-dlss/happy-heron/wiki/State-machine-diagrams, you can do that via `bin/rake state_machines:draw CLASS=Work,Collection FORMAT=svg`

Reply:
I'm working on installing graphviz now...it's been running for a long time.

Reply:
Fun. I can toss a commit at this branch if you'd prefer.

Reply:
That would be fantastic.

Reply:
@jcoyne done

Reply:
@jcoyne last question: should we also make the initial state of the collection `initial` for symmetry?

Reply:
@jcoyne [`3fd31d6` (#844)](https://github.com/sul-dlss/happy-heron/pull/844/commits/3fd31d60b107e0a9a20f72b6969da76e19b2729e)

Reply:
I've updated so that Collection has the same initial state as Work